app/controllers/appointments_controller.rb

```diff
@@ -5,22 +5,27 @@ def index
 	end
 
 	def new
-		@patient = Patient.all
-		@doctor = Doctor.all
+		@patients = Patient.all
+		@doctors = Doctor.all
 		@appointment = Appointment.new
 	end
 
 	def create
 		modified_params= appointment_params.merge(doctor_id: params[:doctor_id], patient_id: params[:patient_id])
 		@appointment=Appointment.new(modified_params)
-		respond_to do |format|
-	      if @appointment.save
-	        format.html { redirect_to admin_index_path, notice: 'Appointment was successfully created.' }
-	        format.json { render :show, status: :created, location: @appointment }
-	      else
-	        format.html { render :new }
-	        format.json { render json: @appointment.errors, status: :unprocessable_entity }
-	      end
+		if Appointment.overload(@appointment.doctor_id,@appointment.schedule)
+			redirect_to admin_index_path, notice: 'Appointment Overloaded'
```

Comment:
have a more general message, something like - 'Sorry, all 15 appointments for today have been booked for Doctor Sanduk Ruit.'